[ECS] Set "strict" was not found

Hello. I'm using "strict" set in my `esc.yml` config and after update from 7.1 to 8.1.20 I faced next error:

```
$ vendor/bin/ecs check --set strict src

                                                                                                                        
 [ERROR] Set "strict" was not found                                                                                     
                                                                                                                        
```

It seems "strict" set missing in [`EasyCodingStandardSetProvider`](https://github.com/symplify/symplify/blob/master/packages/easy-coding-standard/src/Set/EasyCodingStandardSetProvider.php#L21-L39), but it present in [`SetList`](https://github.com/symplify/symplify/blob/master/packages/easy-coding-standard/src/ValueObject/Set/SetList.php#L91)

--- a/packages/easy-coding-standard/src/Set/EasyCodingStandardSetProvider.php
+++ b/packages/easy-coding-standard/src/Set/EasyCodingStandardSetProvider.php
@@ -36,6 +36,7 @@ final class EasyCodingStandardSetProvider implements SetProviderInterface
             SetList::COMMENTS => __DIR__ . '/../../config/set/common/comments.php',
             SetList::DOCBLOCK => __DIR__ . '/../../config/set/common/docblock.php',
             SetList::PHPUNIT => __DIR__ . '/../../config/set/common/phpunit.php',
+            SetList::STRICT => __DIR__ . '/../../config/set/common/strict.php',
         ];
 
         foreach ($setNameToFilePath as $setName => $setFilePath) {

